appholder/src/main/java/com/android/identity/wallet/MainActivity.kt

```diff
@@ -36,6 +44,15 @@ class MainActivity : AppCompatActivity() {
         Navigation.findNavController(this, R.id.nav_host_fragment)
     }
 
+    private lateinit var fusedLocationClient: FusedLocationProviderClient
+    private val presentationLogStore: PresentationLogStore by lazy {
+        ProvisioningUtil.getInstance(applicationContext).logStore
+    }
+
+    object Const {
+        const val LOCATION_REQUEST_CODE = 123
```

Comment:
not really focusing on the back and forth of user denying then granting location permissions, the user is asked once for location permission at the start of the app - if denied then location will be (0,0)